user_guide_src/source/general/errors.rst

```diff
@@ -148,48 +191,48 @@ Since v4.3.0, you can specify the HTTP status code for your Exception class to i
 
 When an exception implementing ``HTTPExceptionInterface`` is caught by CodeIgniter's exception handler, the Exception code will become the HTTP status code.
 
-.. _error-specify-exit-code:
+.. _http-status-code-and-error-views:
 
-Specify Exit Code in Your Exception
-===================================
-
-.. versionadded:: 4.3.0
+HTTP Status Code and Error Views
+================================
 
-Since v4.3.0, you can specify the exit code for your Exception class to implement
-``HasExitCodeInterface``.
-
-When an exception implementing ``HasExitCodeInterface`` is caught by CodeIgniter's exception handler, the code returned from the ``getExitCode()`` method will become the exit code.
+The exception handler displays the error view corresponding to the HTTP status
+code, if one exists.
 
-.. _logging_deprecation_warnings:
+For example, ``PageNotFoundException`` implements the ``HTTPExceptionInterface``,
+so its exception code ``404`` will be the HTTP status code. Therefore if it is
+thrown, the system will show the **error_404.php** in the **app/Views/errors/html**
+folder when it is a web request. If it is invoked via CLI, the system will show
+the **error_404.php** in the **app/Views/errors/cli** folder.
```

Comment:
I think this is the correct behavior, but the current implementation is not.
See  #8689